src/testrel.jl

```diff
@@ -95,14 +95,14 @@ function test_expected(expected::AbstractDict, results, testname::String)
 
         if !isequal(expected_result_tuple_vector, actual_result_vector)
             @warn(
-                "$testname: Expected result vs. actual",
+                "$testname: Expected result vs. actual for $name",
```

Comment:
makes a lot of sense. While we're here... for the `@info` on line 78, could you add the results to that log?